lease/lessor_test.go

```diff
@@ -213,16 +214,19 @@ func TestLessorRenew(t *testing.T) {
 // TestLessorRenewExtendPileup ensures Lessor extends leases on promotion if too many
 // expire at the same time.
 func TestLessorRenewExtendPileup(t *testing.T) {
-	oldRevokeRate := leaseRevokeRate
-	defer func() { leaseRevokeRate = oldRevokeRate }()
-	leaseRevokeRate = 10
```

Comment:
why there can be a lessor routine running when the test code changes leaseRevokeRate? we start the lessor at line 227 after this line. will there be concurrent tests running?